dev/dependencyList

```diff
@@ -77,12 +79,34 @@ jetty-servlet/9.4.41.v20210516//jetty-servlet-9.4.41.v20210516.jar
 jetty-util-ajax/9.4.41.v20210516//jetty-util-ajax-9.4.41.v20210516.jar
 jetty-util/9.4.41.v20210516//jetty-util-9.4.41.v20210516.jar
 jline/0.9.94//jline-0.9.94.jar
+kubernetes-client/5.12.1//kubernetes-client-5.12.1.jar
```

Comment:
we also need to update LICENSE-binary

Reply:
@zwangsheng have you updated LICENSE-binary ? not see the kubernetes-*.jar in LICENSE-binary